types/webpack/index.d.ts

```diff
@@ -920,7 +932,14 @@ declare namespace webpack {
             dependencies: boolean;
         }
 
-        class MainTemplate extends Tapable {}
+        class MainTemplate extends Tapable {
+          hooks: {
+            jsonpScript?: Hook<string, Chunk, string>;
+            requireExtensions: Hook<string, Chunk, string>;
```

Comment:
Should be a `SyncWaterfallHook`: https://github.com/webpack/webpack/blob/ba20513223607eaccb615c8ffabe824413b1613c/lib/MainTemplate.js#L96